projects/sandbox/espresso.md

```diff
@@ -0,0 +1,40 @@
+# Project application: Espresso Logic Minimizer
+
+## Application
+
+### Sandbox application
+
+* Project name: Espresso Logic Minimizer
+* Project repo(s): https://github.com/chipsalliance/espresso/
+* Brief summary of the project: Logic Minimizer Tool, rehost&remaintain under Chips Alliance.
+* Project's open source license: Apache 2.0
```

Comment:
```suggestion
* Project's open source license: BSD Apache 2.0
```
The original Espresso source code doesn't include an explicit license except a few mentions of:
```
Copyright (c) 1988, 1989, Regents of the University of California
```
And as far as I can tell, one more explicit license for part of it:
```
/*
 * Oct Tools Distribution 3.0
 *
 * Copyright (c) 1988, 1989, Regents of the University of California.
 * All rights reserved.
 *
 * Use and copying of this software and preparation of derivative works
 * based upon this software are permitted.  However, any distribution of
 * this software or derivative works must include the above copyright
 * notice.
 *
 * This software is made available AS IS, and neither the Electronics
 * Research Laboratory or the University of California make any
 * warranty about the software, its performance or its conformity to
 * any specification.
 *
 * Suggestions, comments, or improvements are welcome and should be
 * addressed to:
 *
 *   octtools@eros.berkeley.edu
 *   ..!ucbvax!eros!octtools
 */
```
I'm not sure of the best way to handle this but relicensing it is probably out of the question so I think just acknowledging the original license is okay? Will need advice from someone who knows more about this than me, maybe @brianwarner

Reply:
This sounds like a prehistoric BSD license, without third rule:
3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.

Reply:
Project's open source license: BSD Apache 2.0 - is it BSD or Apache 2.0? Or a mix? "BSD Apache 2.0" does not seem to mean anything...

Reply:
I think original license should be licensed with: “Oct Tools Distribution 3.0” and modifications should be license under Apache 2.0. It should be a mix license.
After releasing, we should consider contacting original author to relicense it to Apache or BSD, since “Oct Tools Distribution 3.0” might be too old.

Reply:
Please change the PR text as needed so that the information is correct.